maven-scm-api/src/main/java/org/apache/maven/scm/command/AbstractCommand.java

```diff
@@ -35,7 +35,7 @@
 public abstract class AbstractCommand
     implements Command
 {
-    private ScmLogger logger = new Slf4jScmLogger( getClass() );
+    protected Logger logger = LoggerFactory.getLogger( getClass() );
```

Comment:
better is private and to have separate logger for each class, in this way logger name will be inconsistent with usage.

Reply:
eh, the class hierarchy is really deep, so to make each class has own logger? Meg.
Also, this change above is on par with what we had before: subclasses used getLogger to get logger from parent.... so effect is same sans getter for logger.